src/tests/ftest/util/general_utils.py

```diff
@@ -44,6 +44,60 @@ class DaosTestError(Exception):
     """DAOS API exception class."""
 
 
+def clush(hostlist, command, env=None, verbose=True):
```

Comment:
I wonder if we should make this name more generic so that if we ever replace `clush` in the future with something else, we can just drop that in here and not feel like we need to go updating the name everywhere.

Reply:
Good suggestion.  Let me know what you think of the new name - I'm up for other suggestions if they make sense.

Reply:
How about `pcmd`?

Reply:
Done